data/human/human missions.txt

```diff
@@ -7453,3 +7453,1280 @@ mission "Adelita 5"
+			`	"Sorry man, good luck with your science stuff or whatever. But that's all I know so I'm out of here."`
+			`	Hans waits until you transfer the money over to him, and then quickly retreats deeper into his ship.`
+			`	Tavish watches him go. "I dare say he didn't deserve additional recompense after what he did to us."`
+			`	Kormak grunts in agreement.`
+				accept
+
+
+	on visit
+		dialog `You haven't found the Broken Record yet.`
+	on complete
+		conversation
+			`Your ship lands in the heart of the continent-consuming megatropolis Polyphron, the "only city" on Hephaestus.`
+			`	Finding your way around here could well be impossible. Thankfully Tavish says he knows the area where recently impounded ships are housed.`
+			`	He and Kormak head toward the spaceport.`
+
+
+mission "Tavish and Kormak 4"
+	name "A journey to Chiron"
+	description "Transport Tavish and Kormak to <planet> to look for a man named Doctor-Professor Dylan Hullen-Freeze."
+	source "Hephaestus"
+	destination "Chiron"
+	passengers 2
+	to offer
+		has "Tavish and Kormak 3: done"
+	on offer
+		conversation
+			`	You get the feeling the sun never sets on Hephaestus as you navigate droves of itinerant workers all hurriedly moving from one shift change to next. The number of ships being worked on, landing, taking off, or just sitting is staggering. It would be easy to believe this was the beating heart of the human expansion, and not Earth.`
+			`	Tavish leads you down one narrow corridor filled with endless ships. You don't see any labels or signage that make sense, but somehow Tavish plows on with a quick step and unshakable confidence.`
+			`	Eventually he leads you into a large administrative complex. After four flights of stairs, three different elevators, and one very small trolley the three of you arrive at a bland corporate office with the words, "Repository of Confiscated Goods."`
+			`	There are no doors, and a single window that looks like it was made of reinforced glass. The tired looking agent on the other side of the glass hits a button and his voices buzzes over to you, "What can I help you with?"`
+			choice
+				`	"We're looking for stolen scientific research equipment that was recently confiscated aboard the Broken Record."`
+			`	The man looks down at his monitor and begins typing. After a while he stops and looks back up at you.`
+			`	"Ah yes, I found it. You'll be happy to know we did indeed confiscate some unusual equipment from the Broken Record." He looks at you expectantly.`
+			choice
+				`	"Great! Can it be returned to us?"`
+				`	"Finally. We're here to collect it."`
+				`	"Hand it over. It belongs to us."`
```

Comment:
I am sorry, I don't see the problem here.

Reply:
I'd suggest maybe having the number of choices vary, so it isn't always three, but is somewhere in the 2-4 range. Also, I stand by my earlier statement that you could have a slightly tweaked version of the next paragraph for when the third choice gets selected.

Basically, my argument is that when the three choices are very similar (which they aren't here, so this occurrence actually works to some extent), they may as well be the same choice, at which point the perceived choice is kind of minimized because all the choices are the same.